### PR TITLE
Links

### PR DESCRIPTION
**Pull request by KrasimirRaikov**
index.html with links to a directory, text file, html file and a website.
links

### CODE REVIEW

**Comment by panayotkulchev** [comment]
Could you add pic for each task so it is easier for me to remember which PR for witch task

**Comment by KrasimirRaikov** [comment]
OK

File: links/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>links</title>
+    <link rel="stylesheet" type="text/css" href="linksStyle.css">
+</head>
+<body>
+
+<ul>
+    <li><a href="directory" target="_blank">directory</a></li>
```
Comment (panayotkulchev) [inline links/index.html]: Did you find solution why this directory link did not work last time I visited you.

**Comment by panayotkulchev** [comment]
LGTM!

**Pull request merged**